src/FBAppEventsLogger.ts

```diff
@@ -134,13 +134,13 @@ export type AppEventParam = {
   ValueYes: string;
 };
 
-const {
-  AppEvents,
-  AppEventParams,
-}: {
-  AppEvents: AppEvent;
-  AppEventParams: AppEventParam;
-} = AppEventsLogger.getConstants();
+let AppEvents: AppEvent | undefined = undefined;
+let AppEventParams: AppEventParam | undefined = undefined;
+if (AppEventsLogger !== undefined) {
+  const constants = AppEventsLogger?.getConstants();
+  AppEvents = constants.AppEvents;
+  AppEventParams = constants.AppEventParams;
+}
```

Comment:
const { AppEvents, AppEventParams } = AppEventsLogger?.getConstants() || {};

Reply:
@rnike if you want to make a PR that is cleaner (this looks cleaner...yes) please feel free, I had pretty limited time this weekend and it was breaking one of my pre-commit hooks in a work repo so I just patched as quickly as possible, maybe not as well as possible. Hopefully at least I just didn't break anything :crossed_fingers:

Reply:
Yep don't worry the change you've made is not going to break anything, but I'm still wondering why jest is falling without this change, and why the `6.x.x` is not failing without it. I will make the refactor PR after I figure it out.